lib/Driver/ToolChains.cpp

```diff
@@ -1191,6 +1222,20 @@ addLinkSanitizerLibArgsForDarwin(const ArgList &Args,
       /*AddRPath=*/ shared, TC);
 }
 
+static void addLinkSanitizerLibArgsForWindows(const ArgList &Args,
+                                              ArgStringList &Arguments,
+                                              StringRef Sanitizer,
+                                              const ToolChain &TC,
+                                              bool shared = true) {
+  // Sanitizer runtime libraries requires C++.
+  Arguments.push_back("-lc++");
```

Comment:
This naming is still wrong.  Plus, what if Im using `msvcprt` for the C++ runtime?